corona-warn-app/cwa-app-android
# Prettify svg templates for pdf export (EXPOSUREAPP-13635)

[Ticket](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13635)

Testing:

Scan following EU and non EU certificates:

cwa tc gen -e wru -o --seed 10 --iss AT -s t.0.co=AT --png
cwa vc gen -e wru -o --seed 10 --iss AT -s v.0.co=AT --png
cwa rc gen -e wru -o --seed 10 --iss AT -s r.0.co=AT --png
cwa tc gen -e wru -o --seed 10 --png
cwa vc gen -e wru -o --seed 10 --png
cwa rc gen -e wru -o --seed 10 --png

- Trigger batch export and check if text looks good now
- Export Certificates and check if text looks good now
- Export pdf per Mail and check if text looks good now

Tested in Emulator, Pixel 6 and Samsung A3 where text looked good now
